onboard/utils_test.go

```diff
@@ -122,9 +122,9 @@ func (m *mockCatalogActions) GetPreviousCommit(_ context.Context) (commit *catal
 	return &catalog.CommitLog{Metadata: metadata}, nil
 }
 
-func (m *mockCatalogActions) Commit(_ context.Context, _ string, metadata catalog.Metadata) (*catalog.CommitLog, error) {
+func (m *mockCatalogActions) Commit(_ context.Context, _ string, metadata catalog.Metadata) (string, error) {
```

Comment:
question: why did you change the signature of the return type?

Reply:
It's redundant. Existing and new code only use the ID.